android/src/main/java/com/oney/WebRTCModule/PeerConnectionObserver.java

```diff
@@ -396,8 +396,11 @@ public void onIceCandidate(final IceCandidate candidate) {
         params.putMap("candidate", candidateParams);
         SessionDescription newSdp = peerConnection.getLocalDescription();
         WritableMap newSdpMap = Arguments.createMap();
-        newSdpMap.putString("type", newSdp.type.canonicalForm());
-        newSdpMap.putString("sdp", newSdp.description);
+        // Can happen when doing a rollback.
+        if (newSdp != null) {
```

Comment:
Don't we need to handle this in the JS layer?

Reply:
It's already handled this way on peerConnectionSetLocalDescription on setSuccess in the native layer.
so I just copied that

Reply:
I'm not familiar with objective c, so if you're back can you please fix it on IOS.

Reply:
Ok